### ISSUE TITLE
Cannot read property 'getBoundingClientRect' of undefined  

### ISSUE THREAD

**Issue by renatodeleao** [whitesmith/Flooper.js#24]
# Expected behavior and actual behavior.
To floop

# Steps to reproduce the problem.
import latest npm version

# Stack trace
`Cannot read property 'getBoundingClientRect' of undefined  at Flooper.checkBlocks`

**Comment by pjaneiro** [comment]
@renatodeleao any update here?

**Comment by renatodeleao** [comment]
@pjaneiro nope, i didn't have lazy weekends lately :(. Are you using this for something?
